src/Sylius/Component/Core/OrderProcessing/PaymentProcessor.php

```diff
@@ -43,15 +38,11 @@ public function __construct(FactoryInterface $paymentFactory)
     /**
      * {@inheritdoc}
      */
-    public function createPayment(OrderInterface $order)
+    public function processOrderPayments(OrderInterface $order)
```

Comment:
Hi, I know this was merged a while ago, but I'm just working through an update...

Is this class (or this method) actually doing any processing? 

It seems a bit pointless and could go into the Factory (or a Core extension/decoration of the Factory), because the naming is confusing now. There's no processing of order payments going on here that I can see - just creating them.